packages/default-theme/layouts/default.vue

```diff
@@ -1,33 +1,20 @@
 <template>
   <div>
-    <div id="nav">
```

Comment:
please revert this file, for now, we will have here navigation and probably breadcrumbs

Reply:
moved to another component and used here